mini_pupper_maze/node/ai_image_service.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+from urllib import response
+import rclpy
+from rclpy.node import Node
+import logging
```

Comment:
Please search for **self.get_logger().** in the project and use that logger, which is ROS2's logger